pages/common/cbt.md

```diff
@@ -0,0 +1,28 @@
+# cbt
+
+> Utility for reading data from Google Cloud's Bigtable.
+> More information: <https://cloud.google.com/bigtable/docs/cbt-reference>.
+
+- List tables in current project:
+
+`cbt ls`
+
+- Return count of rows in table:
+
+`cbt count {{table_name}}`
+
+- Read single row from a table with only 1 (most recent) cell revision per column:
+
+`cbt lookup {{table_name}} {{row_key}} cells-per-column={{1}}`
+
+- Read single row with only specified column(s) (omit qualifier to return entire family):
+
+`cbt lookup {{table_name}} {{row_key}} columns={{family1:qualifier1}},{{family2:qualifier2}},{{...}}`
```

Comment:
```suggestion
`cbt lookup "{{table_name}}" "{{row_key}}" columns="{{family1:qualifier1,family2:qualifier2,...}}"`
```

- Mandatory fix: [placeholder format with ellipsis violation](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases): `columns=` doesn't accept two mandatory values and the rest optional ones.
- Optional fix: `{{table_name}}` and `{{row_key}}` and `{{family1:qualifier1,family2:qualifier2,...}}` placeholders can be quoted if they can contain some special characters to POSIX-like shell.

Reply:
good for future reference, wasn't sure if the commas being required affected where the `{{` went